README.org

```diff
@@ -37,6 +37,16 @@ git clone https://github.com/rakanalh/emacs-dashboard ~/.emacs.d/dashboard
   (dashboard-setup-startup-hook))
  #+END_SRC
 
+To add custom items
```

Comment:
This actually wasn't what i was referring to. What i wanted you to add is a documentation on how to customize the existing list rather than creating a new item.

So for example, the current default shows everything, but i only want to show bookmarks or (bookmarks + recent files)... how do i do that? we should have this documented using your approach.